analyses/cell-type-ewings/template_notebooks/clustering-workflow/02-clustering-celltypes.Rmd

```diff
@@ -104,20 +148,28 @@ cluster_classification_df <- all_cluster_results |>
       forcats::fct_infreq() |>
       forcats::fct_relevel("All remaining cell types", after = Inf)
   )
+```
+
 
-cluster_classification_df |> 
-  split(cluster_classification_df$resolution) |> 
-  purrr::map(\(df){cluster_celltype_heatmap(df)}) 
+First, we just look at the cell type assignments on the UMAP. 
 
+```{r fig.height=10, fig.width=10}
+celltype_plot_df <- cluster_classification_df |> 
+  dplyr::select(barcodes, UMAP1, UMAP2, singler_lumped)
+
+plot_faceted_umap(celltype_plot_df, singler_lumped)
 ```
 
-```{r, fig.height=10, fig.width=7}
+To compare results we will calculate the Jaccard similarity index between clusters and cell types. 
+We expect that good clustering will line up with the cell type assignments so that there is ~ 1 cell type per cluster. 
+
+```{r, fig.height=15, fig.width=7}
 # get heatmap for showing cluster assignments vs. singler assignments
 cluster_celltype_heatmap(cluster_classification_df)
 ```
 
 
-## Marker gene expression {.tabset} 
+## Marker gene expression
 
 Finally, we will look at the expression of marker genes across each cluster for each value of `k`. 
```

Comment:
Might be better to refer to this as `nn` instead of `k`, since that's how you've labeled the plots